traffic_ops/app/db/seeds.sql

```diff
@@ -22,6 +22,257 @@ insert into role (id, name, description, priv_level) values (5, 'portal', 'Porta
 insert into role (id, name, description, priv_level) values (6, 'migrations', 'database migrations user - DO NOT REMOVE', 20) ON CONFLICT DO NOTHING;
 insert into role (id, name, description, priv_level) values (7, 'federation', 'Role for Secondary CZF', 15) ON CONFLICT DO NOTHING;
 insert into role (id, name, description, priv_level) values (8, 'steering', 'Role for Steering Delivery Services', 15) ON CONFLICT DO NOTHING;
+insert into role (id, name, description, priv_level) values (9, 'root', 'Role for full capabilities - super-user ', 30) ON CONFLICT DO NOTHING;
```

Comment:
The variety and order looks good to me. However, I'm not sure what would be the implicaitons of changing the priv_level (currently the range is 0-30).